operator/manager/updater.go

```diff
@@ -89,6 +100,12 @@ func (u *updater) do(ctx context.Context, step *core.Step) error {
 		Repo:   repo,
 		Build:  build,
 	}
+
+	logger.
+		WithField("event", payload.Event).
+		WithField("action", payload.Action).
+		Debugln("manager: sending global webhook")
```

Comment:
this logging should probably be added to the `webhook` package, otherwise people that do not have webhooks configured will see these logs, which could cause confusion.

Reply:
Agreed

Reply:
The webhook package doesn't currently have any logging, but I have added it anyway. Let me know if you don't think it fits and I'll (re)move it